libs/internal-sdk/src/models/components/activitynotificationjobresponsedto.ts

```diff
@@ -267,7 +199,7 @@ export type ActivityNotificationJobResponseDto$Outbound = {
     ActivityNotificationExecutionDetailResponseDto$Outbound
   >;
   step: ActivityNotificationStepResponseDto$Outbound;
-  overrides?: ActivityNotificationJobResponseDtoOverrides$Outbound | undefined;
+  overrides?: { [k: string]: any } | undefined;
```

Comment:
will unknown fit here more? 

```suggestion
  overrides?: { [k: string]: unknown } | undefined;
```